### PR TITLE
Added a README

### PR DESCRIPTION
**Pull request by meiyukimccandless**
This pull request includes updates and adjustments to streamline the setup process for the Free Realms server emulator. Key changes include:

- Consolidation of server-side configuration steps.
- Exclusion of client-specific instructions (e.g., character creation resources and `FreeRealms.bat` configuration), as these are now handled within the new OSFR Launcher.

Please review the changes to ensure all crucial aspects of the server setup are covered. Let me know if there are any areas that need additional detail or clarification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/Open-Source-Free-Realms/Sanctuary">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
```
Comment (EDITzDev) [inline README.md:23]: The image is squished, adjust one dimension or use a percentage for both

File: README.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+[forks-url]: https://github.com/Open-Source-Free-Realms/Sanctuary/network/members
+[stars-shield]: https://img.shields.io/github/stars/Open-Source-Free-Realms/Sanctuary.svg?style=for-the-badge
+[stars-url]: https://github.com/Open-Source-Free-Realms/Sanctuary/stargazers
+[issues-shield]: https://img.shields.io/github/issues/Open-Source-Free-Realms/Sanctuary.svg?style=for-the-badge
+[issues-url]: https://github.com/Open-Source-Free-Realms/Sanctuary/issues
+[license-shield]: https://img.shields.io/github/license/Open-Source-Free-Realms/Sanctuary.svg?style=for-the-badge
+[license-url]: https://github.com/Open-Source-Free-Realms/Sanctuary/blob/master/LICENSE.txt
+[linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=for-the-badge&logo=linkedin&colorB=555
+[linkedin-url]: https://linkedin.com/in/linkedin_username
+[product-screenshot]: images/screenshot.png
```
Comment (EDITzDev) [inline README.md:263]: Invalid file extension, it's `screenshot.jpg`

**Review by EDITzDev — CHANGES_REQUESTED** [review]
Looks good so far, minor changes.

Would it be possible for the guides to be moved to the wiki?

File: src/Sanctuary.sln
[the remark this answers is not in the document]
Comment (meiyukimccandless) [reply src/Sanctuary.sln:1]: removed the extra files from solution

File: README.md (hunk 1 above)
Comment (meiyukimccandless) [reply README.md]: fixed the dimensions

File: README.md (hunk 2 above)
Comment (meiyukimccandless) [reply README.md]: thanks for catching that! got it fixed.

**Comment by meiyukimccandless** [comment]
While certain details, such as adjustments for using a standard database instead of a file-based database, should be documented in the wiki, every project should still include a well-crafted README. The README serves as the primary source of information about the project and provides users with a quickstart guide. The wiki, on the other hand, is best suited for covering specific use cases and documenting changes in greater detail.

**Review by EDITzDev — APPROVED** [review]

**Pull request merged**